docs/source/development/linting.md

```diff
@@ -0,0 +1,99 @@
+# Linting
+
+## Introduction
+Linting tools are used to improve the quality of your code by checking for errors and making sure it meets a stylistic
+standard before it is run. As a project grows and goes through various stages of development, it becomes important to
+maintain the code quality. Linting makes your code more readable, easy to debug and maintain, and stylistically
+consistent.
+
+## Set up linting tools
+There are a variety of linting tools available to use with your Kedro projects. This guide shows you how to use
+[`black`](https://github.com/psf/black), [`flake8`](https://gitlab.com/pycqa/flake8), and
+[`isort`](https://github.com/PyCQA/isort) to lint your Kedro projects.
+- **`black`** is a [PEP 8](https://peps.python.org/pep-0008/) compliant opinionated Python code formatter. `black` can
+check for styling inconsistencies and reformat your files in place.
+[You can read more in the `black` documentation](https://black.readthedocs.io/en/stable/).
+- **`flake8`** is a wrapper around [`pep8`](https://pypi.org/project/pep8/),
+[`pyflakes`](https://pypi.org/project/pyflakes/), and [`mccabe`](https://pypi.org/project/mccabe/) which can flag
+programming errors and coding style inconsistencies with respect to [PEP 8](https://peps.python.org/pep-0008/),
+and check the cyclomatic complexity of your code base.
+[You can read more in the `flake8` documentation](https://flake8.pycqa.org/en/latest/).
+- **`isort`** is a Python library used to sort imports alphabetically, and automatically separated into sections and by
```

Comment:
```suggestion
- **`isort`** is a Python library used to sort imports alphabetically and automatically separate them into sections and by
```

Reply:
Made the change in 969c6dc7102657e8a145a83c2de02ae6f8004478! Thanks!